src/main/java/com/example/bookstoreproject/api/book/UserController.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bookstoreproject.api.book;
+
+import com.example.bookstoreproject.domain.user.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.example.bookstoreproject.api.book.UserDTOMapper.toUserDTO;
+import static com.example.bookstoreproject.api.book.UserDTOMapper.toUserDTOs;
+import static com.example.bookstoreproject.domain.user.UserDomainMapper.toUser;
+
+@RestController
+@RequestMapping("/api/v1/users")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+
+    @GetMapping
+    public List<UserDTO> findAll() {
+        return toUserDTOs(userService.findAll());
+    }
+
+    @GetMapping("{id}")
+    public UserDTO findUserById(@PathVariable UUID id) {
+        return toUserDTO(userService.findUserById(id));
+    }
+
+    @PostMapping
+    public UserDTO createUser(final @RequestBody UserDTO userDTO) {
+        return toUserDTO(userService.createUser(toUser(userDTO)));
+    }
+
+    @PutMapping("{id}")
+    public UserDTO updateUser(final @RequestBody UserDTO userDTO, final @PathVariable UUID id) {
```

Comment:
```suggestion
    public UserDTO update(final @PathVariable UUID id), final @RequestBody UserDTO userDTO){
```

Đưa id lên trước vì
- id cần kiểm tra trước
- thống nhất với cách khai báo bên service

Reply:
DONE